drivers/hmis_external_apis/spec/models/hmis_external_apis/external_forms/publish_external_forms_job_spec.rb

```diff
@@ -99,7 +99,7 @@
       site_logo_height: 100,
       recaptcha_key: 'fakekey',
       presign_url: 'http://example.com',
-      csp_content: ["default-src 'self'"],
+      sentry_sdk_url: 'https://js.sentry-cdn.com/your-key.js',
```

Comment:
I think you can use this same DSN key https://github.com/greenriver/hmis-frontend/blob/c37b83c10d89e6bf897e65d8241ad8c0a3eb1c44/.env.production#L3 we'll just need to add the hostname to Sentry. Or we can have devops create a new project in sentry with a new dsn.

Reply:
I added the hostname to sentry already so we should be set.

That env var holds both a DSN and ingestion end-point which is a bit different. It looks like we could make it work by loading sentry from the CDN and configuring it by hand but it doesn't seem like the recommend approach from [the Sentry docs](https://docs.sentry.io/platforms/javascript/install/loader/)

>Sentry supports loading the JavaScript SDK from a CDN. Generally we suggest using our Loader instead.